Yoast/wordpress-seo
# Fix the file editor textarea width after 9971

## Summary

- Restores the default WordPress textarea width on the file editor textarea

## Relevant technical choices:

- after #9971 some CSS rules have now higher specificity; this PR just restores the `99%` width used by WordPress on input and textarea elements with a `large-text` class; for more details please refer to #10066
- this change affects only the Settings pages, including GSC, Social, etc.

## Test instructions

This PR can be tested by following these steps:

- go in Tools > File editor and check the textarea has now a 99% width
- check everything works as expected in all the Settings pages, including GSC, Social, etc. 

/Cc @CarolineGeven 

Fixes #10066
